### PR TITLE
Start development of 1.5.3 as 1.6.0-RC4 @open sesame 6/4 17:09

### PR DESCRIPTION
**Pull request by myungjoo**
As soon as all major known bugs and SVACE/Coverity issues are fixed,
I'm going to release 1.6.0.

Cheers!

```
1.5.2 -> 1.5.3
        - Mediapipe's pipelines may be embedded as an element in NNStreamer pipeline
        - Qualcomm SNPE is supported (tensor-filter subplugin)
        - Verisilicon Vivante is supported (tensor-filter subplugin)
        - NNStreamer --> Protobuf decoder added.

        - Meson script re-worked.
        - Semantics of hardware-acceleration options for tensor-filter re-worked.
        - API/Android: nnfw-runtime (neurun) and SNPE support
        - API/Android: usability update.
        - API/Android: less invokation latency. (more optimization coming in next versions)
        - API/C: bugfixes, architectural upgrade, latency reduction.
        - tensorsink is by default "sync=false". If appsink or tensor_sink in NNStreamer Pipeline API's pipeline has sync=true, emit warning messages.
        - Architectural updates: lower CC, less duplication, removed dependency cycles, less complicated #if statements and blocks.
        - Test suite updates: timeout handling, arm-arch error fixes, more test cases (supplying more negative cases), floating-point handling. and a lot more fixes and performance (latency) optimizations.
        - Build script updates: cleaned up dependencies and applied "feature" meson feature.
        - New tensor-filter subplugin API "v1" released. (with v0 backward compatibility)
        - Tensor-filter now accepts C++ classes as a subplugin. Edge-TPU subplugin is re-written as an example.
        - No more essential class in assert() for optimized binaries.
        - Much less assertions. Apply error-handling instead.
        - Daily build & test activated and published.
        - Rules & policy updated to comply with LF/AI.
        - Memory leaks removed: from demux, split, and a few more components.

        - Knwon bug: edge-TPU subplugin is not working in Tizen devices recently.

```

Signed-off-by: MyungJoo Ham <myungjoo.ham@samsung.com>

### CODE REVIEW

File: CHANGES [hunk 1]
```diff
@@ -1,3 +1,30 @@
+1.5.2 -> 1.5.3
+	- Mediapipe's pipelines may be embedded as an element in NNStreamer pipeline
```
Comment (helloahn) [inline CHANGES:2]: They call it graph.
```suggestion
	- Mediapipe's graphs may be embedded as an element in NNStreamer pipeline
```

File: CHANGES [hunk 2]
```diff
@@ -1,3 +1,30 @@
...
+	- Architectural updates: lower CC, less duplication, removed dependency cycles, less complicated #if statements and blocks.
+	- Test suite updates: timeout handling, arm-arch error fixes, more test cases (supplying more negative cases), floating-point handling. and a lot more fixes and performance (latency) optimizations.
+	- Build script updates: cleaned up dependencies and applied "feature" meson feature.
+	- No more essential class in assert() for optimized binaries.
+	- Much less assertions. Apply error-handling instead.
+	- Daily build & test activated and published.
+	- Rules & policy updated to comply with LF/AI.
+	- Memory leaks removed: from demux, split, and a few more components.
+
+	- Knwon bug: edge-TPU subplugin is not working in Tizen devices recently.
```
Comment (helloahn) [inline CHANGES:26]: a typo here :)

```suggestion
	- Known bug: edge-TPU subplugin is not working in Tizen devices recently.
```

File: CHANGES (hunk 1 above)
Comment (myungjoo) [reply CHANGES]: updated.

File: CHANGES (hunk 2 above)
Comment (myungjoo) [reply CHANGES]: updated.
